### DISCUSSION TITLE
Open Development

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#121] [General & Announcements]
We'd love to talk more about Open Development, per our [recent article on the topic](https://www.blockchaincommons.com/articles/Open-Development/).

What do you agree with? What do you disagree with? What else is needed to created a larger Open Development movement?

**Comment by MikeSWarner** [comment]
Common development needs a common purpose.  Here are a few resources I like on how Cooperation Beats Competition; Pro-Sociality Beats Self-Interest.  Pick a tragedy of the commons and rally support?

Contextually, Neoclassical economics claims that “greed is good”—by following one’s own self-interest, “all boats rise.” Unfortunately, we have evidence—from biology to anthropology to sociology to economics—that it simply isn’t true. Only through pro-social, cooperative behavior can we all thrive. It’s time to put these principles into practice as we collectively tackle the [metacrisis](https://metacrisis.org/META-CRISIS/00.+%F0%9F%91%8B+About/Start+Here).

[Where Do Pro-Social Institutions Come From?](https://evonomics.com/pro-social-institutions-come/) By Pseudoerasmus

[You Might Have Earned It, But Don’t Forget That Your Wealth Came from Society](https://evonomics.com/wealth-of-humans-society-ryan-avent/) by Ryan Avent

[How Norway Proves Laissez-faire Economics Is Not Just Wrong, It’s Toxic.](https://evonomics.com/norway-toxic-trickle-down-david-sloan-wilson/) By David S. Wilson and Dag O. Hessen

[Are We Cooperative or Competitive? Our View Shapes the Economy](https://evonomics.com/are-we-cooperative-or-competitive/) by Sandra Aamodt

[Here Is Why Economics Is Built on a Monumental Mistake](https://evonomics.com/here-is-why-economics-is-built-on-a-monumental-mistake/) by Jag Bhalla

[Parasites, Security, and Conflict: The Origins of Individualism and Collectivism ](https://evonomics.com/a-new-theory-that-explains-economic-individualism-and-collectivism/)by Daniel Hruschka

[Source](https://evonomics.com/covid-19-and-the-next-evolution-of-economics/)
